modules/ROOT/pages/as2/AS2-connector-examples-6-0.adoc

```diff
@@ -0,0 +1,90 @@
+= AS2 - Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Example: Configure a Send AS2 Message
+
+. Place the connector in your flow as applicable.
+. To send an AS2 message with request for MDN in your Mule application, create a global `AS2 Connector Send config` element for use by the AS2 send operations.
+The `AS2 Connector Send config`  provides the following global configuration parameters:
++
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 Partner Name| Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.
```

Comment:
AS2 Partner Name is the field name as displayed in anypoint studio. as2Name is the field name as per a data weave transformation.

Reply:
Added to content.

Reply:
Anypoint Studio is showing Self config - AS2 Partner Name. Therefore it is best to keep this as it is and not change to as2Name. This example is intended for someone who is writing/wiring AS2 connector's operations/sources. So it is best to align this with what they will see in Anypoint Studio.